apps/system/js/app_install_manager.js

```diff
@@ -66,23 +68,13 @@
       this.notifContainer =
               document.getElementById('install-manager-notification-container');
       this.appInfos = {};
+      this.customizationApps = new Set();
```

Comment:
I personally think that this set should be stored in the ftu customization module, and not in app_install_manager. I'd like to keep the amount of FTU code in this file as small as possible. You could then possibly check if an install is migrating by an install like: `Service.query('ftucustomization.isMigrating', manifestURL)`